start.xenit.eu/build.gradle

```diff
@@ -0,0 +1,70 @@
+plugins {
+    id 'java'
+    id 'org.springframework.boot'
+    id 'io.spring.dependency-management'
+    id 'eu.xenit.docker' version '4.0.3'
+    id 'idea'
+}
+
+sourceCompatibility = 1.8
```

Comment:
why is there a sourceCompatibility but not a targetCompatibility?

Reply:
`targetCompatibility` defaults to the value of `sourceCompatibility`

https://docs.gradle.org/current/userguide/java_plugin.html#sec:java_convention_properties